README.md

```diff
@@ -126,6 +126,44 @@ oc patch consoles.operator.openshift.io cluster \
   --patch '{ "spec": { "plugins": ["nmstate-console-plugin"] } }' --type=merge
 ```
 
+
+## Deployment on cluster with Helm
+
+A [Helm](https://helm.sh) chart is available to deploy the plugin to an OpenShift environment.
+
```

Comment:
```
The following Helm parameters are required:

plugin.image: The location of the image containing the plugin that was previously pushed

Additional parameters can be specified if desired. Consult the chart [values](https://github.com/openshift/console-plugin-template/blob/main/charts/openshift-console-plugin/values.yaml) file for the full set of supported parameters.
```

this part is not needed?

Reply:
we don't have a required parameter. `plugin.image`  is not a parameter for us. we just have `image` but its not required

Reply:
can be ignored :)